docs/editor/accessibility.md

```diff
@@ -25,6 +25,46 @@ You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **
 
 When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
 
+## Color vision accessibility
+
+You can search for extensions in **Visual Studio Marketplace** that are compatible for color vision deficiency. Use Extensions `kbstyle(Ctrl+Shift+X)` to access the Marketplace extension search bar and enter "colorblind" to populate relevant options.
+
+![Visual Studio Marketplace in VS Code UI](images/accessibility/accessibility-extension-marketplace.png)
+
+To change the [color theme](/docs/getstarted/themes.md), go to **File** > **Preferences** > **Color Theme** `kbstyle(Ctrl+K)` `kbstyle(Ctrl+T)`.
```

Comment:
Is there a specific color theme that we should recommend depending on the type of color blindness?

Reply:
Thanks, I added a list of recommended extensions for colorblindness.